.github/actions/check-controller-build/check-controller-build.sh

```diff
@@ -22,7 +22,11 @@ if pushd ${crucible_directory}; then
     fi
 
     echo "Setting build-controller to '${build_controller}'"
-    echo "::set-output name=build-controller::${build_controller}"
```

Comment:
were these useful for github debugging ?

Reply:
I'm not sure that it really behaves all that differently in the end, but GitHub seems to think that this is the way forward:

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter